crates/fluvio-version-manager/src/common/version_directory.rs

```diff
@@ -59,6 +60,8 @@ impl VersionDirectory {
 
     /// Sets this version as the active Fluvio Version
     pub fn set_active(&self) -> Result<()> {
+        Self::check_running_processes()?;
```

Comment:
We should have done this check earlier, not after we spent some time and network traffic downloading the artifacts.

Reply:
I second it that it would be nicer to run the check earlier in the process, either first or maybe after the pkgset metadata download (and before the actual downloads).